README.rst

```diff
@@ -175,10 +99,45 @@ More examples:
     # release Netlink socket
     ip.close()
 
-Network namespace examples
---------------------------
+AsyncIPRoute -- asynchronous RTNL API
+-------------------------------------
 
-Network namespace manipulation:
+While `IPRoute` provides a synchronous RTNL API, it is actually build
+around the asyncio-based core.
+
+The same example as above can look like that:
+
+.. code-block:: python
+
+    import asyncio
+
+    from pyroute2 import AsyncIPRoute
+
+    async def main():
+        # get access to the netlink socket
+        ipr = AsyncIPRoute()
```

Comment:
perhaps that a documentation of option (all_ns / ext_ack / strict_check / nlm_echo) could be useful?

Personally I always set ext_ack/strict_check. Error messages are really easier to understand when it's enabled.

Reply:
Ok, will do

Reply:
Added, https://github.com/svinota/pyroute2/blob/p4-docs-prepare/docs/iproute_intro.rst?plain=1